<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fuction</class>
 <widget class="QMainWindow" name="fuction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="accessibleName">
   <string notr="true">function</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-size:24px;
font-family:sans-serif;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background:rgba(0,0,0,0.8);
border-radius:15px;
padding:2px 4px;
border:2px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>70</y>
         <width>441</width>
         <height>541</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QPushButton" name="sysManButton">
          <property name="font">
           <font>
            <family>sans-serif</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border:2px groove gray;
padding:2px 4px;
color:#fff;
border-radius:15px;
background:#03a9f4</string>
          </property>
          <property name="text">
           <string>系统管理</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="userManButton">
          <property name="font">
           <font>
            <family>sans-serif</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border:2px groove gray;
padding:2px 4px;
color:#fff;
border-radius:15px;
background:#03a9f4</string>
          </property>
          <property name="text">
           <string>用户管理</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="dataManButton">
          <property name="font">
           <font>
            <family>sans-serif</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border:2px groove gray;
padding:2px 4px;
color:#fff;
border-radius:15px;
background:#03a9f4</string>
          </property>
          <property name="text">
           <string>数据管理</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="busSearchButton">
          <property name="font">
           <font>
            <family>sans-serif</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border:2px groove gray;
padding:2px 4px;
color:#fff;
border-radius:15px;
background:#03a9f4</string>
          </property>
          <property name="text">
           <string>业务查询</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="busAnaButton">
          <property name="font">
           <font>
            <family>sans-serif</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border:2px groove gray;
padding:2px 4px;
color:#fff;
border-radius:15px;
background:#03a9f4</string>
          </property>
          <property name="text">
           <string>业务分析</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>20</y>
         <width>151</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>sans-serif</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#fff;
background:transparent;
font-size:30px;</string>
       </property>
       <property name="text">
        <string>功能界面</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menutest">
    <property name="title">
     <string>选项</string>
    </property>
    <addaction name="actionexit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionhelp"/>
    <addaction name="actionabout"/>
   </widget>
   <addaction name="menutest"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionload">
   <property name="text">
    <string>登录</string>
   </property>
  </action>
  <action name="actionhelp">
   <property name="text">
    <string>帮助</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionaddfriend">
   <property name="text">
    <string>添加好友</string>
   </property>
  </action>
  <action name="action_app">
   <property name="text">
    <string>&amp;app</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
